jenkins-master/jobs/scripts/workspace/requirements.txt

```diff
@@ -6,8 +6,14 @@ treeherder-client==2.0.1
 # Required by mozinfo
 mozfile==1.2
 
+# https://github.com/mozilla/mozmill-ci/issues/628 (and 831)
+# As long as Python 2.7.10 cannot be used on our master machines
+ndg-httpsclient
+pyasn1
+pyopenssl
```

Comment:
The second requirements file which gets changed in this PR, and which already has those lines. I can pin the versions for sure.